packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.tsx

```diff
@@ -51,9 +54,7 @@ const StarterPromptsCardAction = ({ className, messageBackAction }: Props) => {
       tabIndex={disabled ? -1 : undefined}
       type="button"
     >
-      <div className={classNames['pre-chat-message-activity__card-action-title']}>
-        {'title' in messageBackAction && messageBackAction.title}
-      </div>
+      <div className={classNames['pre-chat-message-activity__card-action-title']}>{title}</div>
```

Comment:
Does the `:empty` selector works with child elements present?

Reply:
No, but if the logic go to this route, it must have at least title and not `:empty`.

What I could do is to add a `pre-chat-message-activity__card-action-box--empty`, but it would almost duplicate `:empty`.

TBH, need more broader thinking how to build blueprint UIs:

1. same component but if-branch
2. same component but empty it out (don't work here because accessibility, can't be `<button>`)
3. different component

I am going option 1 now.

Reply:
Got it!